### DISCUSSION TITLE
24.04

### DISCUSSION THREAD

**Discussion by falkTX** [DISTRHO/Cardinal#643] [Announcements]
This is a feature release that updates to VCV Rack 2.4 and adds quite a few new modules, bringing the module count to 1193

- HostMIDI: Add option to force gaps between MIDI input notes
- Fix build with latest liblo
- Fix Linux VST2 exported symbols
- Fix MindMeldModular in wasm/web builds
- Force-redraw of all child widgets on dark mode change
- Improve Carla/Ildaeil VST3 plugin handling
- Improve Ildaeil plugin bridge handling for macOS
- Improve Ildaeil plugin scanning
- Several fixes for UI size with high-dpi screens
- Stop using cardinal-specific `settings::darkMode`
- Tweak default build flags to better match VCV Rack
- Update base VCV Rack to v2.4
- Use DPF for AU support instead of JUCE
- Use GLES2 render target for Linux ARM release builds

Module updates:
- Add Computerscare
- Add CVfunk
- Add DHE modules
- Add Starling Via
- Update Befaco
- Update GrandeModular
- Update HetrickCV
- Update Sapphire
- Update Surge XT
- Update unless_modules
- Update ValleyAudio

Special note for AU format users: this release breaks backwards compat due to JUCE -> DPF framework change.
You should export your old patches individually, and reload them after updating.

**Comment by TonalPAK** [comment]
I thank you with all my heart for making such incredible technologies available to all people on earth!
Blessings to you and the world!

**Reply by GRK-Astronomer** [reply]
I second @TonalPAK’s comment and gratitude.  I should also thank any and everyone in their effort to virtualize modular synthesis.  I still have to climb out of my Eurorack rabbit hole; but, it is happening despite shifting away from the physical hands-on paradigm.  Cardinal, VCV 2, and Cherry Audio Voltage Modular are helping to break the addiction. 🤣
